dcc-submission-server/src/main/java/org/icgc/dcc/submission/server/sftp/fs/SystemFileHdfsSshFile.java

```diff
@@ -21,13 +21,14 @@
 
 import java.io.IOException;
 
+import org.apache.sshd.common.Session;
 import org.apache.sshd.common.file.SshFile;
 import org.icgc.dcc.submission.server.sftp.SftpContext;
 
 public class SystemFileHdfsSshFile extends BaseDirectoryHdfsSshFile {
 
-  public SystemFileHdfsSshFile(SftpContext context, RootHdfsSshFile root, String directoryName) {
-    super(context, root, directoryName);
+  public SystemFileHdfsSshFile(SftpContext context, RootHdfsSshFile root, String directoryName, Session session) {
+    super(context, root, directoryName, session);
```

Comment:
Can't we just use `root.session` most of the time instead of passing it through the constructor explicitly?